ecosystem/typescript/sdk/src/aptos_types/type_tag.ts

```diff
@@ -236,6 +243,14 @@ function isValidAlphabetic(c: string): boolean {
   return false;
 }
 
+// Generic format is T<digits> - for example T1, T2, T10
+function isGeneric(c: string): boolean {
+  if (c.match(/T\d+/g)) {
+    return true;
+  }
+  return false;
+}
```

Comment:
Hmm, maybe we should just take any generic

Reply:
wdym?